Structure_Engine/Create/SectionProperties/ConcreteSection/ConcreteCircularSection.cs

```diff
@@ -45,12 +45,11 @@ public static partial class Create
         [Input("diameter", "Diameter of the section.", typeof(Length))]
         [Input("material", "Concrete material to be applied to the section. If null a default material will be extracted from the database.")]
         [Input("name", "Name of the concrete section. This is required for most structural packages to create the section.")]
-        [Input("reinforcement", "Optional list of reinforcement to be applied to the section.")]
-        [InputFromProperty("minimumCover")]
+        [Input("rebarIntent", "Optional list of RebarIntent to be applied to the section.")]
         [Output("section", "The created circular concrete section.")]
-        public static ConcreteSection ConcreteCircularSection(double diameter, Concrete material = null, string name = "", List<IBarReinforcement> reinforcement = null, double minimumCover = 0)
+        public static ConcreteSection ConcreteCircularSection(double diameter, Concrete material = null, string name = "", BarRebarIntent rebarIntent = null)
```

Comment:
Not necessarily against this change, but it is not needed really, unless we want to.

Can simply have the methods generating a RebarIntent for you and applying that to the section.

You are working closer to the workflows for this though, so can leave it to you to say what works best script-wise.

Reply:
I guess it depends on our discussion here https://github.com/BHoM/BHoM/pull/1255#discussion_r664513720 about where the `MinimumCover` is stored. 

From a workflow point of view, it's down to how much we thinking `BarRebarIntent` objects will be used as a storage mechanism outside of `ConcreteSection`, and then when assigning it you'd have to explode it for the `Create` method.